### PR TITLE
add safe tx redux actions

### LINKED ISSUE #43: add Safe transaction and setting actions to redux
definition of done:
- [ ] create a tx proposal
- [ ] accept a tx proposal
- [ ] get all txs
- [ ] get safe information

### PR DESCRIPTION
**Pull request by diegoalzate**
closes #43 

### overview
adds redux actions to create, reject and get all transactions. Also get all the safe information.

this pr includes some ui buttons and actions but that should be handled again in another pr

### CODE REVIEW

File: src/sections/safe.tsx [hunk 1]
```diff
@@ -8,21 +7,117 @@ import { useAppDispatch, useAppSelector } from '../store';
 import Section from '../future-hopr-lib-components/Section';
 import { useSigner } from '../hooks';
 import { actionsAsync } from '../store/slices/safe/actionsAsync';
```
Comment (camilovegag) [inline src/sections/safe.tsx:9]: Move this under // Stores

File: src/sections/safe.tsx [hunk 2]
```diff
@@ -8,21 +7,117 @@ import { useAppDispatch, useAppSelector } from '../store';
...
   useEffect(() => {
     if (signer) {
       dispatch(actionsAsync.getSafesByOwnerThunk({ signer }));
     }
   }, [signer]);
+
   return (
     <Section className="Section--safe" id="Section--safe" yellow>
-      SAFE REDUX STORE
+      <h1>SAFE REDUX STORE</h1>
```
Comment (camilovegag) [inline src/sections/safe.tsx:24]: Just a comment, when we work on the UI we might want to handle uppercase text with styles `text-transform: uppercase;`  :)

File: src/sections/safe.tsx (hunk 1 above)
Comment (diegoalzate) [reply src/sections/safe.tsx:9]: i can change this in this pr but it was not part of the existing changes.

File: src/sections/safe.tsx (hunk 2 above)
Comment (diegoalzate) [reply src/sections/safe.tsx:24]: yeah the uppercase was also not part of this pr

File: src/sections/safe.tsx [hunk 3]
```diff
@@ -1,28 +1,123 @@
...
     if (signer) {
       dispatch(actionsAsync.getSafesByOwnerThunk({ signer }));
     }
   }, [signer]);
+
   return (
     <Section className="Section--safe" id="Section--safe" yellow>
-      SAFE REDUX STORE
+      <h1>Safe redux store</h1>
       <pre>{JSON.stringify(safe, null, 4)}</pre>
```
Comment (mjadach-iv) [inline src/sections/safe.tsx:25]: Please switch around the store and actions, so `<pre>{JSON.stringify(safe, null, 4)}</pre>` is below actions

**Review by mjadach-iv — CHANGES_REQUESTED** [review]
If a wallet is not connected in the UI, the `develop/safe` shouldn't work

**Review by mjadach-iv — APPROVED** [review]

**Pull request merged**